Dashboard/Models/StepData.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Dashboard.Models;
+
+/// <summary>
+///     The data class for each step of the flowchart.
+/// </summary>
+public class StepData
+{
+    public string Step { get; set; } = string.Empty;
```

Comment:
You could probably make this a record as its just a data class, also could consider adding the required keyword to each of these to avoid needing to set default instantiations

Reply:
6d30670d3dd57630e69139a3b1a94ba94b62176e